mmdet/models/roi_heads/cascade_roi_head.py

```diff
@@ -317,13 +335,14 @@ def simple_test(self, x, proposal_list, img_metas, rescale=False):
                         self.bbox_head[i].loss_cls.get_activation(s)
                         for s in cls_score
                     ]
-                bbox_label = [s[:, :-1].argmax(dim=1) for s in cls_score]
-                rois = torch.cat([
-                    self.bbox_head[i].regress_by_class(rois[j], bbox_label[j],
-                                                       bbox_pred[j],
-                                                       img_metas[j])
-                    for j in range(num_imgs)
-                ])
+                refine_roi_list = []
+                for j in range(num_imgs):
+                    if rois[j].shape[0] > 0:
+                        bbox_label = cls_score[j][:, :-1].argmax(dim=1)
+                        refine_roi = self.bbox_head[i].regress_by_class(
+                            rois[j], bbox_label[j], bbox_pred[j], img_metas[j])
+                        refine_roi_list.append(refine_roi)
```

Comment:
1. It seems that rois.shape==0 is not handled, will that have influence?
2. refine_roi -> refined_rois